composer.json

```
@@ -12,13 +12,13 @@
         }
     ],
     "require": {
-        "php": ">=5.3.2",
-        "symfony/framework-bundle": "~2.7|~3.2|~4.0",
+        "php": "^7.1.3",
+        "symfony/framework-bundle": "~4.3|~5.0",
```

```suggestion
        "symfony/framework-bundle": "^4.3 || ^5.0",
```

To uniformize with the `require-dev` section.